sql/db-connection.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+function addElective($title, $description, $lecturer){
+
+	$host = "localhost";
+	$db = "db_name";
+	$username = "root";
+	$pass = "";
+
+	$conn = new PDO("mysql:host=$host;dbname=$db;charset=utf8", $username, $pass);
```

Comment:
Currently if I try to use the DB and I use a wrong password to connect to it, the information is not being recorded. It is failing silently without showing any indications to the user about the reason. Could we adapt the code in such a way that the user knows if the connection to the DB was not successful?